bad-bits/test/bad-bits.test.js

```diff
@@ -103,11 +102,6 @@ describe('fetchAndStoreBadBits', () => {
       'If-None-Match in the first request',
     ).toBe(null)
 
-    // The history is updated when new bad bits are fetched
-    expect(await getBadBitsHistory(env)).toStrictEqual([
-      { timestamp: lastUpdateAt.toISOString(), etag },
-    ])
```

Comment:
I guess we don't need to keep an audit trail of all bad-bit updates we performed. However, I think we should at least keep the timestamp of when we finished the last full update. To show that we are actively keeping up with bad-bits updates.